packages/pipeline/test/data_sources/ohlcv_external/crypto_compare_test.ts

```diff
@@ -13,7 +13,7 @@ const expect = chai.expect;
 describe('ohlcv_external data source (Crypto Compare)', () => {
     describe('generateBackfillIntervals', () => {
         it('generates pairs with intervals to query', () => {
-            const source = new CryptoCompareOHLCVSource();
+            const source = new CryptoCompareOHLCVSource(20);
```

Comment:
Jw why you use a different limit here than the default 15 per second

Reply:
aah no reason. I also removed defaults from the class itself. Decided it was better to set defaults in just one place (entrypoint script) to be more obvious.